src/Installers/Windows/HostOptions/Product.wxs

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+	<Product Id="$(var.ProductCode)" Name="$(var.ProductName)" Language="1033" Version="$(var.Version)" 
+			 Manufacturer="Microsoft Corporation" UpgradeCode="$(var.UpgradeCode)">
+		<Package InstallerVersion="$(var.InstallerVersion)" Compressed="yes" InstallScope="perMachine" />
+
+		<MajorUpgrade DowngradeErrorMessage="A newer version of [ProductName] is already installed." />
+		<Media Id="1" />
```

Comment:
Entire file has bizarre indentation. Please align bits at the same level and avoid tabs.